src/lib.rs

```diff
@@ -9,7 +9,7 @@ use core::ops::FnOnce;
 /// Trying to set the same fields twice will generate a compile-time error. Trying to build without
 /// setting one of the fields will also generate a compile-time error - unless that field is marked
 /// as `#[builder(default)]`, in which case the `::default()` value of it's type will be picked. If
-/// you want to set a different default, use `#[builder(default=...)]`.
+/// you want to set a different default, use `#[builder(default=…)]`.
```

Comment:
Were these created automatically by something?

Reply:
oh, I think I did a search and replace

Reply:
I noticed that you used the unicode character and wanted to do that to what I added as well

Reply:
Didn't even notice it was in the rest of the docs until now. Turns out it was added in https://github.com/idanarye/rust-typed-builder/commit/3a748906c8f22ae0dfba74dcb8b24917a00da851.

After this gets in, I'm going to search-and-replace it with `...`. No reason to deviate from ASCII just for this...